projects/utils-maps/src/lib/ol/utils-ol.ts

```diff
@@ -75,7 +75,11 @@ export function mapToSingleCanvas(map: Map, targetCanvas: HTMLCanvasElement | Of
                 // Step 3: copy source bitmap to target-canvas.
                 targetContext.drawImage(sourceCanvas, 0, 0, sourceCanvas.clientWidth, sourceCanvas.clientHeight, 0, 0, targetCanvas.width, targetCanvas.height);
             });
-            subscriptions.push(key);
+            if (Array.isArray(key)){
```

Comment:
>> `if (Array.isArray(key)){`

Wasn't aware that a subscription-key can be an array. Thanks for catching that. Did this cause any issues anywhere?

Reply:
The typescript compiler made me aware of this :)
I don't think that it made any issues, but compiler errors should be fixed.